packages/cli/src/createProject.fixtures.ts

```diff
@@ -40,7 +41,10 @@ async function getExampleProject(networkFamily: string, network: string): Promis
     code: string;
     networks: {code: string; examples: ExampleProjectInterface[]}[];
   }[];
-  return templates.find((t) => t.code === networkFamily)?.networks.find((n) => n.code === network)?.examples[0];
+  const template = templates.find((t) => t.code === networkFamily)?.networks.find((n) => n.code === network)
+    ?.examples[0];
+  assert(template, 'Failed to get template');
```

Comment:
I think this function can return undefined, its a change in behaviour i don't think is needed

Reply:
However, cloneProjectTemplate() requires exampleProject to be defined.